### PR TITLE
[ci] Fix awscurl run headers

### PR DESCRIPTION
**Pull request by xyang16**
## Description ##

Brief description of what this PR is about

Fix awscurl run headers: Remove space because it's string.

### CODE REVIEW

File: tests/integration/llm/client.py [hunk 1]
```diff
@@ -975,7 +975,7 @@ def awscurl_run(data,
                 json_results=False,
                 random_delay=False):
     find_awscurl()
-    headers = "Content-type: application/json"
+    headers = "Content-type:application/json"
```
Comment (frankfliu) [inline tests/integration/llm/client.py:978]: Why this causing problem?

Comment (xyang16) [reply tests/integration/llm/client.py:978]: See this example:

`./awscurl -c 1 -N 5 -X POST http://127.0.0.1:8080/invocations --connect-timeout 300 -H Content-type: application/json -d '[{"key": "what is panda?", "value": "hi"}]'   -P -t`

Having space in this the Content-type not passed in correctly. The other fix is add quote for Content-type.

Comment (frankfliu) [reply tests/integration/llm/client.py:978]: I see, we should add quote instead of remove space

Comment (xyang16) [reply tests/integration/llm/client.py]: Changed.

**Review by frankfliu — APPROVED** [review]

**Pull request merged**